imagelab/WindowCloser.java

```diff
@@ -1,29 +1,41 @@
 package imagelab;
-import imagelab.*;
-import java.awt.*;
-import java.awt.event.*;
-import java.awt.image.*;
+
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
 
 /**
  * Used to close an active window.
+ *
  * @author Dr. Aaron Gordon
  * @author Dr. Jody Paul
  */
 public class WindowCloser extends WindowAdapter {
 
     /** The image frame to be killed. */
-    ILFrame theFrame;
+    private ILFrame theFrame;
 
-    public WindowCloser(ILFrame f) {
+    /**
+     * Frame Constructor.
+     * @param f The Frame
```

Comment:
If this is a "frame constructor" then why is a "frame" being passed in as the parameter?